docs/serving/istio-authorization.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Enabling requests to Knative services when additional authorization policies are enabled"
+weight: 25
+type: "docs"
+---
+
+Knative Serving system pods, such as the activator and autoscaler components, require access to your deployed Knative services.
+If you have configured additional security features, such as Istio's authorization policy, you must enable access to your Knative service for these system pods.
+
+## Before you begin
+
+You must meet the following prerequisites to use Istio AuthorizationPolicy:
+
+- [Enabling istio sidecar injection](https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/).
```

Comment:
Two things:

1. How about "Istio sidecar injection must be enabled"?

2. Does it matter whether the sidecar injection is manual or automatic?

Reply:
> How about "Istio sidecar injection must be enabled"?

Yes, updated to  "Istio sidecar injection must be enabled".

> Does it matter whether the sidecar injection is manual or automatic?

No, it does not matter. I did not mention it in the doc, but should I? I think it is no problem without mentioning about the manual injection or automatic injection.